app/controllers/reviewControllers.js

```diff
@@ -71,10 +72,25 @@ const patchReviewById = (request, response, next) => {
     });
 };
 
+const deleteCommentById = (request, response, next) => {
+  const comment_id = request.params.comment_id;
+  return fetchCommentByCommentId(comment_id)
```

Comment:
I assume you have added this first so that if the comment ID doesn't exist you get a promise reject?
If so you could also write a promise reject into the removeCommentByID model as PSQL would give an error if there was no ID and it would save two calls to the db